### PR TITLE
feat: Implemented Switch (#384)

### PR DESCRIPTION
**Pull request by johglove**

### CODE REVIEW

File: src/components/PageContent/Switch/Switch.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      aria-checked={on}
+      aria-label={label}
+      className={classNames(classNameArr)}
+      onClick={handleClick}
+      role="switch"
+      {...(testMode && { "data-testid": TestUtils.Switch.container })}
+      {...attrs}
+    >
+      <span class="rvt-switch__on">On</span>
+      <span class="rvt-switch__off">Off</span>
```
Comment (pcberg) [inline src/components/PageContent/Switch/Switch.jsx:61]: className instead of class in React, right?

File: src/components/PageContent/util/TestUtils
Comment (pcberg) [inline src/components/PageContent/util/TestUtils:1]: Is this file an accident?

File: src/components/PageContent/Switch/Switch.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  const variantOptions = variant ? Array.isArray(variant) ? variant : [variant] : []
+  const variantClasses = variantOptions.map(value => {
+    switch(value) {
+      case "danger":
+        return "rvt-switch--danger"
+      case "small":
+        return "rvt-switch--small"
+      default:
+        return ""
+    }
```
Comment (pcberg) [inline src/components/PageContent/Switch/Switch.jsx:42]: You could replace this with
`variant ? `rvt-switch--${variant}` : "",`

File: src/components/PageContent/Switch/Switch.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  /** An unseen label for the menu to help with accessibility */
+  label: PropTypes.string.isRequired,
+  /** Supplement function ran when switch is clicked, parameters (newSwitchState, clickEvent) */
+  onClick: PropTypes.func,
+  /** Initial on/off state of the switch */
+  startOn: PropTypes.bool,
+  /** [Developer] Adds data-testId attributes for component testing */
+  testMode: PropTypes.bool,
+  /** The variant type which determines how the switch is styled. Options include: [danger, small]*/
+  variant: PropTypes.oneOfType([PropTypes.string,  PropTypes.array]),
```
Comment (pcberg) [inline src/components/PageContent/Switch/Switch.jsx:76]: Replace with
`PropTypes.oneOf(["danger", "small"])
?

**Review by pcberg — CHANGES_REQUESTED** [review]

File: src/components/PageContent/Switch/Switch.jsx (hunk 3 above)
Comment (johglove) [reply src/components/PageContent/Switch/Switch.jsx:76]: Variant can be a single value ("small", "danger") or multiple values (["small", "danger"])

**Review by pcberg — APPROVED** [review]

**Pull request merged**